Domain/Models/Auth/User.cs

```diff
@@ -16,5 +16,9 @@ public class User
         [Required] public string Password { get; set; }
 
         public ICollection<UserRole> UserRoles { get; set; } = new Collection<UserRole>();
+        
+        
+        public int? AzureInformationId { get; set; }                    
```

Comment:
`int?` Ka slags triks e det her?

Reply:
int? betyr nullable int. dvs. at `AzureInformationId `kan holde på ingenting, `NULL`.

https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/nullable-value-types

Reply:
Det Janton sa. Der e tilfelle der User ikkje he AzureInfo, og da må id'en kunne være null

Reply:
Takk for godt svar!